core/templates/services/compute-graph.service.ts

```diff
@@ -25,7 +25,7 @@ import { States } from 'domain/exploration/StatesObjectFactory';
 export interface GraphLink {
   source: string;
   target: string;
-  linkProperty?: string;
+  linkProperty: string | null;
```

Comment:
Why is `null` allowed here?

Reply:
I will handle this while strict typing this file. This file is not yet fully strict typed.

Reply:
GraphLink has been used between different files, sometimes linkProperty have type string and some time it is not defined,  it has to be corrected in all the files as jeevesh will makes  all the file strict null will automatically remove,